Documentation/ignition.md

```diff
@@ -18,9 +20,9 @@ Reference an Ignition config in a [Profile](bootcfg.md#profiles). When PXE booti
 
 ## Configs
 
-Here is an example Ignition template for static networking, which will be rendered, with metadata, into YAML and tranformed into machine-friendly JSON.
+Here is an example Ignition template for static networking. This template will be rendered into YAML, using metadata (from a Group) to fill in the template values. Once templated the YAML is then tranformed into standard JSON. All of this happens at query time, so that when Ignition queries bootcfg the only thing it sees is the final standard JSON output.
 
-ignition/network.tmpl:
+ignition/network.yaml:
```

Comment:
possibly use `network.yaml.tmpl` which is something Crawford and I have discussed.